applications/RANSApplication/custom_python/rans_python_application.cpp

```diff
@@ -43,6 +43,53 @@ PYBIND11_MODULE(KratosRANSApplication,m)
     AddCustomStrategiesToPython(m);
     AddCustomUtilitiesToPython(m);
     AddCustomProcessesToPython(m);
+
+    // incompressible potential flow specific variables
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, VELOCITY_POTENTIAL )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESSURE_POTENTIAL )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RANS_IS_INLET )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RANS_IS_OUTLET )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RANS_IS_STRUCTURE )
```

Comment:
Can't the `INLET`, `OUTLET` and `STRUCTURE` flags be used for this purpose? (note that these can now be synchronized among MPI processes).

Reply:
I create copies of same model part having same geometries but different conditions and elements. But all of them needs to have the same boundary flags. So I copy `INLET` like boundary flags to their respective data value containers since those containers are stored inside geometries. So I can have consistent boundary flags across my all of duplicated model parts. If we can have a flag container also in geometries, that would be the best.